lms/envs/minimal.yml

```
@@ -36,3 +36,6 @@ LMS_INTERNAL_ROOT_URL: "http://localhost"
 
 # So that Swagger config code doesn't complain
 API_ACCESS_MANAGER_EMAIL: "api-access@example.com"
+
+# So that you can login to studio on bare-metal
+SOCIAL_AUTH_EDX_OAUTH2_URL_ROOT: 'http://localhost:18000'
```

```suggestion
SOCIAL_AUTH_EDX_OAUTH2_URL_ROOT: 'http://localhost:18000'
SOCIAL_AUTH_EDX_OAUTH2_KEY: 'cms-sso'
```

as mentioned above -- since we are telling them to use `cms-sso` as the client ID anyway, why not provide it here for them too?

---

Disregard-- we'll use `studio-sso-id` for now in order to remain compatible with devstack.py